manifests/rbac.yaml

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
I like the separation of rbac as a separate manifest 👍

Reply:
👍